### PR TITLE
fix v1 trap varBinds

### PR DESCRIPTION
**Pull request by apsivam**
Work around to correctly forward v1 TRAP varBinds

### CODE REVIEW

File: scripts/snmpfwd-server.py [hunk 1]
```diff
@@ -209,9 +209,13 @@ def processPdu(self, snmpEngine, messageProcessingModel,
 
             if messageProcessingModel == 0:
                 pdu = rfc2576.v1ToV2(pdu)
+                #get varBinds from original trap
+                orig_varbinds = v2c.apiTrapPDU.getVarBinds(pdu)
 
                 # TODO: why this is not automatic?
                 v2c.apiTrapPDU.setDefaults(pdu)
```
Comment (etingof) [inline scripts/snmpfwd-server.py]: Do we really need `setDefaults` call here at all? Could we get away with just:

    pdu = rfc2576.v1ToV2(pdu)

It seems `v1ToV2()` alone would fully initialize v2c PDU.

Comment (apsivam) [reply scripts/snmpfwd-server.py]: I guess we do as v1ToV2 doesn't add request ID, error status & error index to the converted v2c PDU.

Comment (etingof) [reply scripts/snmpfwd-server.py]: Right, but are those values needed for TRAPs? They seem to be [explicitly ignored](https://github.com/etingof/pysnmp/blob/master/pysnmp/proto/proxy/rfc2576.py#L161) in pysnmp which is coded close to respective SNMP RFC (e.g. RFC-2576).

**Comment by etingof** [comment]
Let me merge this patch and do a followup so we have this resolved.

File: scripts/snmpfwd-server.py (hunk 1 above)
Comment (etingof) [reply scripts/snmpfwd-server.py]: I'd appreciate your testing of my fix up commit 0a3b819d4f20f64d538973cc001b14cead6167b1. Thanks! ;-)

**Pull request merged**